src/background.js

```diff
@@ -390,12 +391,20 @@ function main() {
       )
 
       let isLastestVersion = existConfigFile && existWitnetFile
+      let isLatestVersionCompatible = true
+
       if (existVersionFile) {
         try {
           const versionFile = fs.readFileSync(
             path.join(SHEIKAH_PATH, VERSION_FILE_NAME),
             'utf8',
           )
+          const diff = semver.diff(
+            getVersionFromName(versionFile),
+            getVersionFromName(latestReleaseVersion),
+          )
+          isLatestVersionCompatible = !diff || diff === 'patch'
+
```

Comment:
```suggestion
          isLatestVersionCompatible = semver.satisfies(getVersionFromName(latestReleaseVersion), `~${getVersionFromName(versionFile)}`)

```

Reply:
Using ~1.2.1 will make version 1.2.1 compatible with 1.2.2, but not with 1.3.0 and not with 2.0.0. Is that what we want?

Reply:
I think so